packages/ui/src/documents/DocumentForm.tsx

```diff
@@ -91,9 +100,33 @@ export class DocumentForm extends React.Component<Props, State> {
       },
     };
 
-    this.props.onSubmit && this.props.onSubmit(payload);
+    onSubmit && onSubmit(payload);
   };
 
+  addCollaboratorToPayload = (collaborators: Array<any>) => {
+    const { document } = this.props;
+    let read_access = [];
+    let write_access = [];
+
+    if (collaborators.length > 0) {
+      collaborators.forEach(c => {
```

Comment:
`forEach` should already take care of the 0-length array. I think we can drop the collaborators.length check